data/Sankhadip-Roy.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Sankhadip Roy",
+  "displayStatsPublic": true,
```

Comment:
We no longer need this `displayStatsPublic` property. So please remove this from your `json`.

```suggestion
```